app/models/miq_alert.rb

```diff
@@ -76,6 +76,9 @@ def self.assigned_to_target(target, event = nil)
     # Get all assigned, enabled alerts based on target class and event
     cond = "enabled = ? AND db = ?"
     args = [true, target.class.base_model.name]
+    if target.class == ManageIQ::Providers::Openshift::ContainerManager
+      args = [true, "ContainerManager"]
+    end
```

Comment:
don't remember why here I set "ContainerManager" but left base "ExtManagementSystem" in line below,
but apparently it works